src/board-explorer-app/board-explorer-app.html

```diff
@@ -1387,13 +1387,17 @@
 
     getDocumentation: function(item, type) {
       return new Promise(function(resolve, reject) {
-        if (item.html) {
-          return resolve({
-            status: 200,
-            content: item.html
-          });
+
+        if (!item.html) {
+          console.warn("Additional documentation for " + item.refdes + " was not found.");
+          item.html = "";
         }
```

Comment:
@juliocri This needs to be and `} else {`  to return resolve() otherwise this.$.boardViewer.getDocumentation() isn't called.